src/comlink.ts

```diff
@@ -295,6 +295,9 @@ export function expose(
   ep: Endpoint = globalThis as any,
   allowedOrigins: (string | RegExp)[] = ["*"]
 ) {
+  const newCount = (proxyCounter.get(ep) || 0) + 1;
+  proxyCounter.set(ep, newCount);
```

Comment:
A few notes:
* I think `proxyCounter` should be renamed. Maybe `endpointRefCounts` ?
* We might want to return from `Comlink.expose` a function that decrement that count (so that this can be cleaned)